### PR TITLE
added system role and better shell recognition for windows

### PR DESCRIPTION
**Pull request by parsapoorsh**
- Added .gitignore file
- Added system role for better quality
- If the program is opened on Windows cmd, it will be recognized.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
+    author="Farkhod Sadykov | parsa poorsh",
+    author_email="parsa.poorsh@gmail.com",
     description=(
         "A command-line productivity tool powered by ChatGPT, "
         "will help you accomplish your tasks faster and more efficiently."
     ),
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
-    url="https://github.com/ther1d/shell_gpt",
+    url="https://github.com/parsapoorsh/shell_gpt",
```
Comment (TheR1D) [inline setup.py:25]: Nice try, banned.

Comment (TheR1D) [inline setup.py:25]: I guess it was intended to use in forked PYPI

**Comment by TheR1D** [comment]
I would suggest to keep this changes as a fork.

**Comment by TheR1D** [comment]
Hey @parsapoorsh I didn't notice there is cool streaming stuff in the PR, and closed it, becasue of changes in setup, my bad. Please feel free to open another PR (can't reopen this one).

**Comment by parsapoorsh** [comment]
> Hey @parsapoorsh I didn't notice there is cool streaming stuff in the PR, and closed it, becasue of changes in setup, my bad. Please feel free to open another PR (can't reopen this one).

here you are :smile:  https://github.com/TheR1D/shell_gpt/pull/107

**Pull request closed without merging**